_dev/apps/ui/src/App.vue

```diff
@@ -38,6 +38,7 @@
           </MenuItem>
           <MenuItem
             :route="{name: 'Billing'}"
+            v-if="GET_BILLING_SUBSCRIPTION_ACTIVE"
```

Comment:
Is this variable a flag ? a boolean ?
if it is, it should be better to have a name that reflects it, as isGetBillingSubActive or hasGetBillingSub....

Reply:
It's a boolean, and thanks to the `mapGetter` the name of the getter we can directly use without proxy variable.

Reply:
but where is it defined ?

Reply:
This is likely because I'm not aiming for `master` here. Were you on the branch `dev/monetization`?

Reply:
oh. ok, different branch.

Reply:
No it's still scoped in the component. `mapGetters` is a helper able to use the store available in the component to return the getters provided in an array.

Reply:
ok, now I got it.
so, no, it's not a boolean, is an object's method.
and I reckon this code is only testing if the method is defined or not.
Now, it raises the question why do we need to test whether it's defined.....
I reckon it should be always defined, no ?
Or is this available behind a feature flag ?

Reply:
`mapGetters` will make the name of the getter always available to the component make it return the result of the associated method in https://github.com/PrestaShopCorp/psxmarketingwithgoogle/blob/ba7c19b415adc5480ab5b7b71080376f8e8296c9/_dev/apps/ui/src/store/modules/app/getters.ts#L121-L125

The type of `GET_BILLING_SUBSCRIPTION_ACTIVE` is always a boolean is this line.

Reply:
ok. so back to my initial comment :
this getter name, as it is a boolean, should have a name that shows it; like
IS_BILLING_SUBSCRIPTION_ACTIVE
or
HAS_BILLING_SUBSCRIPTION

Reply:
De ce que je comprends, GET_BILLING_SUBSCRIPTION_ACTIVE est bien une méthode "(LocalState): boolean" qui est appelée au runtime

Reply:
nope. c'est un getter vuex.
C'est un peu le même principe pour une prop en js. Exemple:
```javascript
const MyObject = {
    firstname: "Terry",
    lastname: "Berger",
    get name() {
        return this.firstname + "_"+ this.lastname
    }
};

console.log( myObject.name );
// Terry_Berger
```